### ISSUE TITLE
help text does not show [subvolume...] argument for run/dryrun actions

### ISSUE THREAD

**Issue by diraimondo** [digint/btrbk#21]
I would keep in a unique configuration file all the remote/local subvolume managed by the backup server. Such volumes could be managed using different strategies (once per day, once per week, only sync because snapshots are made remotely using recent introduced features, ...). It would nice if I could restrict on command-line the selected action on a specified subvolume (not all). As long as I understand, it is not possible at the moment: all the subvolumes are involved.

**Comment by digint** [comment]
This is possible, btrbk takes [subvolume...] arguments for run/dryrun action.
Suppose your config looks like this:
```
volume /mnt/btr_pool
  subvolume  data
...
volume ssh://my.remote.host/mnt/btr_pool
  subvolume remote_data
...
```
then you can run:
```
btrbk dryrun /mnt/btr_pool/data
btrbk dryrun ssh://my.remote.host/mnt/btr_pool/remote_data
```
and btrbk will skip all other volume/subvolume declarations (with lots of `!!!` in the summary for each skipped subvolume)

**Comment by diraimondo** [comment]
That's what I suspected but the output of the help text should be improved. Thanks.

**Comment by digint** [comment]
Yep, it's missing in the help text. It's in the man-page though. Will fix this in the next revision.

**Comment by digint** [comment]
Fixed in v0.19.0, with much cleaner output
